examples/chat-online-status/react/README.md

```diff
@@ -24,35 +24,35 @@ Find out more about [online status](/docs/chat/rooms/presence?lang=react).
 
 1. Clone the [Ably docs](https://github.com/ably/docs) repository where this example can be found:
 
-  ```sh
```

Comment:
Not sure if this was your linter, but if these aren't tabbed then the numbering resets after every code block.

Reply:
It was my linter. I'll revert these and add a note to the repo for prettier to get its paws off these readmes